utils/src/main/java/uk/gov/moj/sdt/utils/mbeans/SdtMetricsMBean.java

```diff
@@ -33,7 +33,6 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.stereotype.Component;
```

Comment:
I'm going to assume there is a good reason to change this and it won't break any autoconfiguration.

Reply:
Not breaking this, as this is a utility class has static method to create instance, better to use that than @Component annotation